docs/streams/developer-guide/config-streams.html

```diff
@@ -333,7 +333,7 @@ <h4><a class="toc-backref" href="#id5">bootstrap.servers</a><a class="headerlink
                           context.taskId(), record.topic(), record.partition(), record.offset(),
                           exception);
 
-                      dlqProducer.send(new ProducerRecord&lt;&gt;(dlqTopic, record.timestamp(), record.key(), record.value()));
+                      dlqProducer.send(new ProducerRecord&lt;&gt;(dlqTopic, record.timestamp(), record.key(), record.value(), record.headers())).get();
```

Comment:
Sorry for being picky. What if `.get()` throws an exception? Or is it good enough as-is for an example?